docs/Flux_Functions.md

```diff
@@ -0,0 +1,10 @@
+# Flux functions
+
+Create new functions in Flux code. 
+
+## Required guidelines
+
+- Flux functions must conform to examples in [universe.flux](https://github.com/influxdata/flux/blob/master/stdlib/universe/universe.flux).
+- You must submit a unit test in [testing](https://github.com/influxdata/flux/tree/master/stdlib/testing).
```

Comment:
should be an end-to-end test in  stdlib/testing/testdata.

you should add another short doc  `adding an end to end test in flux`